src/reference/90-Developers-Guide/02-Coding-Guideline/00.md

```diff
@@ -90,83 +90,87 @@ Use [MiMa][mima].
 
 #### Public traits should contain `def` declarations only
 
+TODO: "results to codegen at subclass" is unclear
```

Comment:
Thanks for the review Dale!

I need to understand the technical point being made better in order to suggest improved English, but if the intended meaning of "results to codegen" is close to "causes code generation", then I would typically say "results *in*", not "results to" (or "causes").

I *think* I would write something like:

> `val`s or `var`s defined in `trait`s result in code generation when extended, and artificial `Foo\$class.\$init\$` compilation units.

I'm not certain "compilation unit" is the correct term.

Also, it might help to distinguish clearly how `def` differs—is the point that `val` and `var` produce synthesized accessor/modifier property methods for their underlying fields?